frontend/src/components/home-map/HomeMap.js

```diff
@@ -39,10 +39,15 @@ const colorRange = ["#EB5353", "#F9D923", "#9ACD32", "#36AE7C"];
 const higlightColor = "#84b4cc";
 const isMarker = false;
 
-const HomeMap = ({ current, style }) => {
+const HomeMap = ({ current, style, mapValues }) => {
```

Comment:
We can clean up this HomeMap (disregard the unused calculation, Markers, etc, and use map data from `Home.jsx` only as you wrote here).